lms/djangoapps/support/views/enrollments.py

```diff
@@ -176,11 +185,12 @@ def get_course_modes(course_key):
             list of `Mode`
 
         """
-        course_modes = CourseMode.modes_for_course(
```

Comment:
I believe we should modify `CourseMode.modes_for_course()` to work both ways instead of copying the logic here.